sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/DownloadRetryOptions.java

```diff
@@ -12,10 +12,10 @@
  * This class contains the configuration options used to reliably download from the blob service.
  */
 @Fluent
-public final class ReliableDownloadOptions {
+public final class DownloadRetryOptions {
     private static final String PARAMETER_NOT_IN_RANGE = "The value of the parameter '%s' should be between %s and %s.";
```

Comment:
Message can simply be `Max retries cannot be a negative integer.` 
Current message prints out `"The value of the parameter '%s' should be between 0 and 2147483647."`. Not very user-friendly.

Reply:
I'm happy to make this change if others approve

Reply:
@srnagar Lets do this in a follow-up issue.